samples/main.cpp

```diff
@@ -181,14 +184,18 @@ int main(int /*argc*/, char* /*argv*/[])
 
   auto loggingLevel = NovelRT::LogLevel::Debug;
 
-  memeInteractionRect->Interacted += [&console] {
+  memeInteractionRect->Interacted += [&] {
     console.logDebug("WAHEYYY");
+    audio.lock()->playSound(jojo, 0);
+    audio.lock()->playSound(jojo2, 0);
+    audio.lock()->playSound(jojo3, 0);
+    audio.lock()->playSound(jojo4, 0);
   };
 
   //If uncommenting the call, pass &jojo to the subscription next to &audio.
```

Comment:
Nit: comment no longer relevant

Reply:
This needs resolving before I'll approve.